Tests/ValetIntegrationTests/KeychainIntegrationTests.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  KeychainIntegrationTests.swift
+//  Valet iOS
+//
+//  Created by Dan Federman on 5/20/20.
+//  Copyright © 2010 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import XCTest
+
+@testable import Valet
+
+
+class KeychainIntegrationTests: XCTestCase {
+
+    func test_revertMigration_removesAllMigratedKeys() throws {
+        guard testEnvironmentIsSigned() else {
+            return
```

Comment:
Should we `XCTFail` inside here to make it clear that the test didn't actually run and pass?

Reply:
This test will always run. We'll hit this `return` when we run Mac tests in CI. That is expected.